### ISSUE TITLE
log_softmax potential issue in tflite int8 quantization (how can we skip it?)

### ISSUE THREAD

**Issue by gyin-ai** [tensorflow/tensorflow#48816]
Since tf.nn.log_softmax givens unexpected constant [-15.9375] for many dimensions. Can I ask how we can skip particular operations quantization?

**Comment by abattery** [comment]
@gyin-ai Please share simple stand alone code such that we can replicate the issue faced or if possible share a colab gist.

**Comment by gyin-ai** [comment]
https://colab.research.google.com/drive/17-UsPhrBmGkJGk3b4fYH1-voCrkYw4Vz?usp=sharing 
@abattery

**Comment by abattery** [comment]
@jianlijianli could you triage this issue?

**Comment by jianlijianli** [comment]
Hi gyin-ai,

I noticed you are using random numbers as the "representative_dataset". This is reasonable for making sure the conversion works but accuracy might be impacted. representative_dataset is to run some "representative" data (usually a subset of actual data) through the system so the min/max information is captured for activations and quantization process uses the min/max information to quantize the state.

Could you please use some real data for representative_dataset?

For skipping particular operations quantization, we are working on some prototype of API that allows selectively quantize (or skip quantize) certain ops/layers. @daverim, @liufengdb, could you please help with the API? Thanks.
